presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestMySqlIntegrationSmokeTest.java

```diff
@@ -248,11 +241,8 @@ public void testColumnComment()
     }
 
     private void execute(String sql)
-            throws SQLException
+            throws Exception
```

Comment:
Why change this? `Exception` is good for `@Test` method but I'd prefer if helper methods throw the correct exception type.